src/components/common/Navbar/Navbar.tsx

```diff
@@ -13,37 +13,38 @@ import { mark } from 'src/sdk/tests/mark'
 import { useModal } from 'src/sdk/ui/modal/Provider'
 import type { AnchorHTMLAttributes } from 'react'
 import type { SearchInputRef } from '@faststore/ui'
-import type { StoreCollectionQuery } from '@generated/graphql'
 
 interface NavLinksProps {
   onClickLink?: AnchorHTMLAttributes<HTMLAnchorElement>['onClick']
 }
 
-function NavLinks({ onClickLink }: NavLinksProps) {
-  const {
-    allStoreCollection: { edges: links },
-  } = useStaticQuery<StoreCollectionQuery>(graphql`
-    query StoreCollection {
-      allStoreCollection(filter: { type: { eq: Department } }) {
-        edges {
-          node {
-            slug
-            seo {
-              title
-            }
-          }
-        }
-      }
-    }
-  `)
+const collections = [
+  {
+    name: 'Office',
+    href: '/office',
+  },
+  {
+    name: 'Home Appliances',
+    href: '/kitchen-and-home-appliances',
+  },
+  {
+    name: 'Computer and Software',
+    href: '/computer-and-software',
+  },
+  {
+    name: 'Technology',
+    href: '/technology',
+  },
+]
```

Comment:
out of curiosity: why these mocked collections? should it come as props?

Reply:
we don't have those allCategories queries anymore :(